src/Nox.Generator/Domain/EntityMeta.template.cs

```diff
@@ -40,4 +41,16 @@ public partial class {{className}}
         => Nox.Types.{{entityMetaData.Type}}.From(value);
     {{ end }}
 {{- end }}
+
+    {{- for attribute in entity.Attributes }}
+
+        /// <summary>
+        /// User Interface for property '{{attribute.Name}}'
+        /// </summary>
+        public static TypeUserInterface? {{attribute.Name}}UserInterface(NoxSolution solution) 
+            => solution.Domain?
+                .Entities?.FirstOrDefault(e => e.Name == "{{entity.Name}}")?
```

Comment:
use GetEntityByName will be more efficient

Reply:
updated